src/Sql/Sql/help/New-AzSqlServerTrustGroup.md

```diff
@@ -0,0 +1,139 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/powershell/module/az.sql/new-azsqlservertrustgroup
+schema: 2.0.0
+---
+
+# New-AzSqlServerTrustGroup
+
+## SYNOPSIS
+Creates a new Server Trust Group.
+
+## SYNTAX
+
+```
+New-AzSqlServerTrustGroup [-ResourceGroupName] <String> [-Location] <String> [-Name] <String>
+ [-GroupMembers] <System.Collections.Generic.List`1[System.String]>
+ [[-TrustScope] <System.Collections.Generic.List`1[System.String]>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates a new Server Trust Group with specified location, members, trust scope, name and resource group.}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> New-AzSqlServerTrustGroup -Location "West Europe" -Name "ServerTrustGroup01" -GroupMembers "ManagedInstance01","ManagedInstance02" -TrustScope "GlobalTransactions" -ResourceGroupName "ResourceGroup01" 
```

Comment:
When specifying -GroupMembers we need more information than just a name of managed instance (as MIs can be from different resource groups, locations and subscriptions - we cannot inherit those information from STG).

We can possibly provide here the full resourceId of a group member, or retrieve that information as part of PS command. Not sure what's the common approach here.